### PR TITLE
#RI-5134 - add test connections panel

### PR DESCRIPTION
**Pull request by AmirAllayarovSofteq**

### CODE REVIEW

File: redisinsight/ui/src/pages/rdi/pipeline/components/test-connections-log/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    .euiCollapsibleNavGroup__children {
+      padding: 0 !important;
+    }
+
+    .euiIEFlexWrapFix {
+      flex-grow: 1;
+      padding-right: 4px;
+    }
+
+    //.euiAccordion-isOpen .euiAccordion__triggerWrapper {
```
Comment (kyle-marcum) [inline redisinsight/ui/src/pages/rdi/pipeline/components/test-connections-log/styles.module.scss:73]: did you mean to leave this comment?

File: redisinsight/ui/src/pages/rdi/pipeline/pages/config/Config.spec.tsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 import { useFormikContext } from 'formik'
 import { rdiPipelineSelector } from 'uiSrc/slices/rdi/pipeline'
-import { fireEvent, cleanup, render, screen } from 'uiSrc/utils/test-utils'
+import { rdiTestConnectionsSelector } from 'uiSrc/slices/rdi/testConnections'
+import {fireEvent, cleanup, render, screen, act} from 'uiSrc/utils/test-utils'
```
Comment (kyle-marcum) [inline redisinsight/ui/src/pages/rdi/pipeline/pages/config/Config.spec.tsx:5]: Screenshot 2024-02-14 at 11 09 36 PM

File: redisinsight/ui/src/pages/rdi/pipeline/components/test-connections-log/styles.module.scss (hunk 1 above)
Comment (AmirAllayarovSofteq) [reply redisinsight/ui/src/pages/rdi/pipeline/components/test-connections-log/styles.module.scss:73]: Thank you, i removed it

**Review by kyle-marcum — APPROVED** [review]

**Comment by kyle-marcum** [comment]
See this comment here: https://www.figma.com/file/z08rwfM11zwq2Mu5W0qjgJ?node-id=708:41167&mode=design#702747376

It's small but I figured could be updated in this PR

**Pull request merged**